File: api/Models/ProducerData.cs
namespace api.Models
{
    public class ProducerData
    {
        public int Id { get; set; }
        public string Description { get; set; } = string.Empty;
        public string Name { get; set; } = string.Empty;
        public string Profession { get; set; } = string.Empty;
        public byte[]? Image_One { get; set; } = null!;
        public byte[]? Image_Two { get; set; } = null!;
        public byte[]? Image_Three { get; set; } = null!;
        public byte[]? Image_Profile { get; set; } = null!;
        public string? ImageUrl { get; set; } = string.Empty;
        public int UserId { get; set; }
        public User? User { get; set; }
    }
}

File: webApp/src/app/shared/models/producerData.ts
import { User } from "./user";

export class ProducerData {
    id: number = 0;
    description: string = '';
    name: string = '';
    profession: string = '';
    image_One: string = '';
    image_One_Url: string = '';
    image_Two: string = '';
    image_Two_Url: string = '';
    image_Three: string = '';
    image_Three_Url: string = '';
    image_Profile: string = '';
    image_Profile_Url: string = '';
    userId: number = 0;
    user: User = new User();
}
